src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,241 @@
+
+### Conditions
+
+Modules from the same chunks and cache group will form a new chunk when all conditions are fulfilled.
+
+There are 4 options to configure the conditions:
+
+* `minSize` (default: 30000) Minimum size for a chunk.
+* `minChunks` (default: 1) Minimum number of chunks that share a module before splitting
+* `maxInitialRequests` (default 3) Maximum number of parallel requests at an entrypoint
+* `maxAsyncRequests` (default 5) Maximum number of parallel requests at on-demand loading
+
+### Naming
+
+To control the chunk name of the split chunk the `name` option can be used.
+
+W> When assigning equal names to different split chunks, all vendor modules are placed into a single shared chunk, though it's not recommend since it can result in more code downloaded.
+
+The magic value `true` automatically chooses a name based on chunks and cache group key, otherwise a string or function can be passed.
+
+When the name matches an entry point name, the entry point is removed.
+
+#### `optimization.splitChunks.automaticNameDelimiter`
+
+By default webpack will generate names using origin and name of the chunk, like `vendors~main.js`.
+
+If your project has a conflict with the `~` character, it can be changed by setting this option to any other value that works for your project: `automaticNameDelimiter: "-"`.
+
+Then the resulting names will look like `vendors-main.js`.
+
+### Select modules
+
+The `test` option controls which modules are selected by this cache group. Omitting it selects all modules. It can be a RegExp, string or function.
+
+It can match the absolute module resource path or chunk names. When a chunk name is matched, all modules in this chunk are selected.
+
+### Select chunks
+
+With the `chunks` option the selected chunks can be configured.
+
+There are 3 values possible `"initial"`, `"async"` and `"all"`. When configured the optimization only selects initial chunks, on-demand chunks or all chunks.
+
+The option `reuseExistingChunk` allows to reuse existing chunks instead of creating a new one when modules match exactly.
+
+This can be controlled per cache group.
+
+### `optimization.splitChunks.chunks: all`
+
+As it was mentioned before this plugin will affect dynamic imported modules. Setting the `optimization.splitChunks.chunks` option to `"all"` initial chunks will get affected by it (even the ones not imported dynamically). This way chunks can even be shared between entry points and on-demand loading.
+
+This is the recommended configuration.
+
+T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/), it will inject all the generated vendor chunks for you.
+
+## `optimization.splitChunks`
+
+This configuration object represents the default behavior of the `SplitChunksPlugin`.
+
+```js
+splitChunks: {
+	chunks: "async",
+	minSize: 30000,
+	minChunks: 1,
+	maxAsyncRequests: 5,
+	maxInitialRequests: 3,
+  automaticNameDelimiter: '~',
```

Comment:
I think the indentation is wrong here. It looks like everything else is using tabs, but this line uses spaces. 
<img width="575" alt="screen shot 2018-03-30 at 5 44 50 pm" src="https://user-images.githubusercontent.com/195534/38158042-14ff613e-3442-11e8-8fd7-780433a7be98.png">

Reply:
Thanks for the catch!